test/helpers/deployZNS.ts

```diff
@@ -195,6 +227,11 @@ export const deployZNS = async ({
     registrar,
   };
 
+  // Final configuration steps
+  // TODO AC: remove all redundant calls here! and delete hashing of the root and the need
+  // for Registrar to be owner/operator of the root
+  await registry.connect(deployer).setOwnerOperator(registrar.address, true);
```

Comment:
why do we need this call here exactly? this seems to go against our AC structure.

Reply:
This was in place over a month ago before we had AC and when we still had subdomains. None of the contracts use this and the tests aren't reliant on it anymore so it was just redundant code, I removed it